app/components/UI/TemplateRenderer/SafeComponentList.ts

```diff
@@ -26,6 +26,7 @@ import { SnapUISelector } from '../../UI/Snaps/SnapUISelector/SnapUISelector';
 import InfoRow from '../../Views/confirmations/components/UI/InfoRow';
 import { SnapUISpinner } from '../../Snaps/SnapUISpinner/SnapUISpinner';
 
+import { SnapUITooltip } from '../../Snaps/SnapUIToolTip/SnapUIToolTip';
```

Comment:
```suggestion
import { SnapUITooltip } from '../../Snaps/SnapUIToolTip/SnapUITooltip';

```
Can we fix the casing and spacing here?

Reply:
Fixed this.